components/BridgeActionsPair.tsx

```diff
@@ -69,8 +73,11 @@ const BridgeActionsPair: VFC<
 	useEffect(() => {
 		if (!fromOption) return;
 		if (fromOption === "Ethereum") return updateMetaMaskBalances?.();
-		if (fromOption === "CENNZnet") return updateCENNZBalances?.();
-	}, [fromOption, updateMetaMaskBalances, updateCENNZBalances]);
+		if (fromOption === "CENNZnet") {
+			void updateCENNZBalances?.(setBalances);
```

Comment:
could have the hook `useUpdateCENNZBalances` getting `setBalances` internally, that way you can keep the `updateCENNZBalances?.()` signature same as it was (without passing any parameters)